pg_logplexcollector.go

```diff
@@ -32,7 +32,19 @@ func listen(die dieCh, sr *serveRecord) {
 		os.Remove(sr.P)
 		pc, err = net.ListenPacket("unixgram", sr.P)
 	case "logfile":
-		f, err = os.Open(sr.P)
+		i := 0
+		for {
+			f, err = os.Open(sr.P)
+			if err != nil {
+				if i < 15 {
```

Comment:
I don't see where `i` is ever incremented. Unless that is done outside of this patch somehow?